apm-agent-plugins/apm-kafka-plugin/apm-kafka-headers-plugin/src/main/java/co/elastic/apm/agent/kafka/KafkaProducerHeadersInstrumentation.java

```diff
@@ -66,26 +55,23 @@ public KafkaProducerHeadersInstrumentation(ElasticApmTracer tracer) {
 
     @Override
     public String getAdviceClassName() {
-        return "co.elastic.apm.agent.kafka.KafkaProducerHeadersInstrumentation$KafkaProducerHeadersAdvice";
+        return getClass().getName() + "$KafkaProducerHeadersAdvice";
     }
 
     @SuppressWarnings("rawtypes")
```

Comment:
[minor] can't we remove the `@SuppressWarnings("rawtypes")`here while we are at it ?